fs/walk.py

```diff
@@ -85,11 +87,22 @@ def __init__(
                 any of these patterns will be removed from the walk.
             filter_dirs (list, optional): A list of patterns that will be used
                 to match directories paths. The walk will only open directories
```

Comment:
Would it make sense to change "directories paths" here to "directory names"? :shrug: